examples/SerialIface/SerialIface.ino

```diff
@@ -85,21 +95,36 @@ void processBinaryCmds() {
   }
 }
 
+void processFastBinaryCmds() {
+  while (Serial.available() >= FAST_BINARY_CMD_SIZE) {
+    byte cmd[FAST_BINARY_CMD_SIZE];
+    Serial.readBytes(cmd, FAST_BINARY_CMD_SIZE);
+
+    if (RESET_CMD != cmd[0]) {
+      opl2.write(cmd[0], cmd[1]);
+    } else {
+      opl2.init();
```

Comment:
Maybe init can be called when you receive the buffer command? Then you're only passing through serial data.

Reply:
The reset command is usually sent before closing to not let the OPL2 running with a tune.

Reply:
Good point, I probably should have called it `reset` or `reinit` in the original protocol.